string name
int32 space
bool useSensorValues
---
float32[6] position
